packages/react-native-select-pro/src/components/options-list/options-list.tsx

```diff
@@ -62,6 +60,9 @@ export const OptionsList = ({
         sectionHeaderTextStyle,
         containerStyle,
     } = optionsListStyles ?? {};
+    const optionsWithSections = isSectionOptionsType(optionsData);
```

Comment:
maybe: `isSectionOptionsData` or `isOptionsWithSections` , when we have boolean we should try to use words with `is`